docs/_static/theme_override.css

```diff
@@ -0,0 +1,16 @@
+/* Table text wrapping CSS fix from
+https://rackerlabs.github.io/docs-rackspace/tools/rtd-tables.html */
+
+/* override table width restrictions */
+@media screen and (min-width: 767px) {
```

Comment:
Just out of curiosity, why don't we want the wrapping behavior on the small size?

Reply:
I don't know but I can guess. Perhaps the intention is that on small size screens such as smartphones, the column size would be so small and the text would wrap so much that it would be difficult to read. 

I'm not attached to this behavior, since I just used the CSS from the tutorial. I think the ideal behavior would be that below a certain width, the table doesn't shrink horizontally any more (so that at small screen sizes, there would be some scrolling but not too much). Current behavior is in screenshots below:

On a laptop:

![laptop screenshot](https://user-images.githubusercontent.com/10187488/38842213-9e46ab9c-419d-11e8-8882-024a69666c85.png)

On a smartphone:

![phone screenshot 1](https://user-images.githubusercontent.com/10187488/38842232-b93fe0b2-419d-11e8-9881-0b0dc89f1de4.png)

![phone screenshot 2](https://user-images.githubusercontent.com/10187488/38842241-c6755a5a-419d-11e8-8984-bf3c9ba955df.png)